src/hooks/todayTile/useTodayTasks.tsx

```diff
@@ -2,16 +2,37 @@ import { useCallback } from 'react';
 import { useGraphQLClient } from '../useGraphQLClient';
 import { gql } from 'graphql-request';
 import { useActiveAccount } from '../useActiveAccount';
-import { useQuery } from '@tanstack/react-query';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
 import { useActiveProject } from '../useActiveProject';
-import { useRestQuery } from '../rest-api';
+import { useRestQuery, useRestCache } from '../rest-api';
 import { ConsentTask, SurveyResponse } from './types';
 
 export type TodayTask = SurveyResponse | ConsentTask;
 export function isConsentTask(value: TodayTask): value is ConsentTask {
   return !!(value as ConsentTask).form;
 }
 
+export const useInvalidateTodayCountCache = () => {
+  const restCache = useRestCache();
+  const queryClient = useQueryClient();
+
+  // We wait 3.5 seconds by default to give the backend a chance to update
```

Comment:
I'd think that there wouldn't be this long of a delay between when we've submitted a survey and  when the data is available on the backend. I also assumed that the user would maybe take a second or two before navigating directly back to the Home Screen. Did you find this to be an issue in your testing?

1) submitting a survey or completing some activity (submit to the backend)
2) closing the activity and navigating away from the today applet and back to the Home Screen

Reply:
Yeah my testing comprised of completing an activity (user interaction, not a survey) and then navigating back as fast as I could. I saw it reliably work with 3.5 but I agree that is pretty arbitrary and could depend on a variety of factors. Maybe it would be worth assuming a 1-2 second delay on the today tile before navigating back. Open to suggestions.

Reply:
If it works reliably and the task count change feels responsive enough on the home screen then 3.5 seconds is fine with me.